packages/projects/default-project/apartment.scene.json

```diff
@@ -13,11 +13,7 @@
           "name": "fog",
           "props": {
             "type": "linear",
-            "color": {
-              "r": 1,
-              "g": 1,
-              "b": 1
-            },
+            "color": {},
```

Comment:
color properties requires to hold numerical value, may be `0x000000` is accurate

Reply:
this is because the fog node does not serialize properly. in another PR i will re-create all the scenes so we can be assured the data for all the scenes is correct